waiter/src/waiter/handler.clj

```diff
@@ -254,34 +254,57 @@
     :scale-amount
     utils/scale-amount->scaling-state))
 
+(defn param-value->filter-fn
+  "Creates the filter function that does substring match for the input string or any string in the sequence."
```

Comment:
This description is unclear. Maybe something like:

"Takes a single string or a sequence of strings. Returns a filter function ...."

But then, that description makes this smell like it should be split into two functions  strs->filter-fn .... which handles the multiple case (the second branch of the if) and then this functon. Or maybe after splitting, inline this function into its caller?

Reply:
Created `strs->filter-fn`